backend/src/controllers/user.ts

```diff
@@ -1,7 +1,44 @@
 import { NextFunction, Request, Response } from 'express'
 
-export const getUser = ( req: Request, res: Response, next: NextFunction) => {
-  const { userid } = req.params
+import User from 'models/user'
+
+export const getUser = async ( req: Request, res: Response, next: NextFunction ) => {
+  const { uid } = req.params
 
   // Retrieve user by id
+  try {
+    const user = await User.findById(uid).orFail()
+    return res.status(200).json({
+      user
+    })
+  } catch (error) {
+    return res.status(500).json({
```

Comment:
Same problem here, we prob don't want a 500 if the user doesn't exist. Just returning null is prob fine.